src/components/Tab.react.js

```diff
@@ -18,7 +18,7 @@ Tab.propTypes = {
     /**
      * The tab's label
      */
-    label: PropTypes.children,
+    label: PropTypes.string,
```

Comment:
Support for components/something else than a string for label will have to come later, anything else than a string doesn't really work now anyways, so best to set it to string for now.